apps/betterangels-backend/clients/tests/test_mutations.py

```diff
@@ -681,6 +690,77 @@ def test_delete_client_document(self) -> None:
             "The document should have been deleted from the database.",
         )
 
+    def test_update_client_document_success(self) -> None:
+        """Test successful update of a client document's filename."""
+
+        self._handle_user_login("org_1_case_manager_1")
+
+        new_filename = "updated_document_name.txt"
+        document_id = self.client_profile_1_document_1["id"]
+
+        response = self.execute_graphql(
+            """
+                mutation UpdateClientDocument($data: UpdateClientDocumentInput!) {
+                    updateClientDocument(data: $data) {
+                        ... on ClientDocumentType {
+                            id
+                            originalFilename
+                        }
+                    }
+                }
+            """,
+            variables={
+                "data": {
+                    "id": document_id,
+                    "originalFilename": new_filename,
+                }
+            },
+        )
+
+        self.assertIsNotNone(response.get("data"))
+        self.assertIsNotNone(response["data"].get("updateClientDocument"))
+        self.assertEqual(response["data"]["updateClientDocument"]["originalFilename"], new_filename)
+
+        # Verify database was updated
+        document = Attachment.objects.get(id=document_id)
+        self.assertEqual(document.original_filename, new_filename)
+
+    def test_update_client_document_empty_filename(self) -> None:
```

Comment:
I don't think we need this test. Technically, I don't think there's anything in the requirements stating that a filename can't be empty